### ISSUE TITLE
one gpu training

### ISSUE THREAD

**Issue by RaymondCover** [ohhhyeahhh/SiamCAR#17]
I only get one 2080ti for training， I find that in config.yaml the batch size is setted as 80, so I change it to 28.  Should I adjust the learning rate or change something else. Did this change influnence the results a lot?

**Comment by ohhhyeahhh** [comment]
> I only get one 2080ti for training， I find that in config.yaml the batch size is setted as 80, so I change it to 28. Should I adjust the learning rate or change something else. Did this change influnence the results a lot?

Hi, I am sorry to say, We have not done this comparison experiment before, May be you can have a try by keeping other paras unchanged. Please let me know when you get the results.

**Comment by RaymondCover** [comment]
1. I keep other paras unchanged and find that the batch size really influenced the results,  maybe I should reduce the lr?
2. I also find that after unfix the backbone params,  I didn't get a better perforamance, could you let me konw you can get your best results at which epoch?  
3. When I use multi-Gpus, can I just set ''os.environ["CUDA_VISIBLE_DEVICES"] = '0,1,2,3' '' in train.py?
Really thanks for your help

**Comment by mr-xiao-zuo** [comment]
> 1. I keep other paras unchanged and find that the batch size really influenced the results,  maybe I should reduce the lr?
> 2. I also find that after unfix the backbone params,  I didn't get a better perforamance, could you let me konw you can get your best results at which epoch?
> 3. When I use multi-Gpus, can I just set ''os.environ["CUDA_VISIBLE_DEVICES"] = '0,1,2,3' '' in train.py?
>    Really thanks for your help

Hi, Did you change the learning rate after changing batch processing? What advice do you have for training the code with the Tesla P40 8G of a single graphics card? Thanks.
